sdkconfig.defaults

```diff
@@ -18,7 +18,8 @@ CONFIG_APP_OFFSET=0x10000
 
 # Bump the stack size of the Weave task to accommodate extra stack useage due to
 # debugging.
-CONFIG_WEAVE_TASK_STACK_SIZE=5120
+# 8192 is requied to avoid stackoverflow
```

Comment:
Wasn't 5120 similarly required? Rather than putting a magic number in here for the comment, it might be better for future readers to indicate what functionality caused this to increase such that if future maintainers are working on that call path and things crash, they might correlate the two.

Reply:
After some debug using GDB, find the cause of stackoverflow, now using a 6144B(=6KB) stack size for WEAVE task, tested (create fabric-provisioning via wifi-pairing) for a while without stackoverflow found, should be enough for ESP3.3. The previous 8192B is to increase the size of stack to a big enough value, which is a bit too large. And 6144B is more suitable.